### ISSUE TITLE
Review and document deployment process

### ISSUE THREAD

**Issue by robvanmieghem** [threefoldfoundation/tfchain#188]
Heavily coupled with issue #182. That issue will be resolved while reviewing and documenting the deployment process.

Currently  only Geert and Zaibon can deploy the blockcreators, this is not a good situation.

**Comment by zaibon** [comment]
extremely bad indeed

**Comment by GlenDC** [comment]
Was already planned implicitly as part of https://github.com/threefoldfoundation/tfchain/issues/182, but not really tracked nor made clear in that issue.

However the end goal is:

- we know exactly what our scripts do and (can) maintain them;
- we know good enough how our environments function;
- the entire deployment of all (and a subset of all) nodes is documented and fully automated;
- the required environment en toolset (including versions) is also documented in complete detail;

All this should be enough to allow any member of the team who has the correct credentials available locally to deploy to testnet and standard net for all (or a subset of all) nodes.

Transfering the ownership of nodes from zaibon/geert to us was already discussed with @zaibon in PM, but not tracked yet, thanks for doing so @robvanmieghem.

Was planning to do so the next milestone (or whenever js has a stable version).

**Comment by zaibon** [comment]
I think once you have #182 done and under control, I should meet with you @GlenDC and do some training/handover so you can have a clear view on how things looks in the field. So from there you can create a plan to migrate everything

**Comment by GlenDC** [comment]
Yeah that sounds good, once I know how it all works from learning on my own as well as your training/handover, I will formalise the process as documentation for our project. Once that documentation is there (after the training/handover and after I wrote it), you can also probably best review it, as you are familiar with the environment and deployment process in general (not project specific, but still, should allow you to spot things that should be different or could be better).
